src/program/lwaftr/tests/data/icmp_on_fail.conf

```diff
@@ -1,3 +1,4 @@
+name lwaftr1;
```

Comment:
I think this will lead to spurious test failures, as this file is used by many tests.  We wouldn't want to prevent multiple people from running "lwaftr check" at once.